Client/Frontend/Browser/HomePanel/NTPDownloader.swift

```diff
@@ -380,6 +506,53 @@ class NTPDownloader {
                 }
             }
             
+            if let customTheme = item as? CustomTheme, let topSites = customTheme.topSites {
+                /// For favicons we do not move them to temp directory but write directly to a folder with favicon overrides.
+                guard let saveLocation =
+                    FileManager.default.getOrCreateFolder(name: NTPDownloader.faviconOverridesDirectory) else {
+                        throw "Failed to create directory for favicon overrides"
+                }
+                
+                for topSite in topSites {
```

Comment:
@iccub - The spec states that:

`Return an HTTP status code of 200 (i.e. no redirects allowed).` - I don't think we are doing that here. Let me know if I'm missing something.

Reply:
I missed this: https://github.com/brave/brave-ios/blob/9cebfb5556cd1962f860ac7761b9ff94664a4d09/Client/Frontend/Browser/HomePanel/NTPDownloader.swift#L437 

Resolved 👍

Reply:
We check for network codes here

https://github.com/brave/brave-ios/blob/9cebfb5556cd1962f860ac7761b9ff94664a4d09/Client/Frontend/Browser/HomePanel/NTPDownloader.swift#L437